product/pom.xml

```diff
@@ -6,10 +6,19 @@
         <artifactId>mapstore-root</artifactId>
         <version>1.8-SNAPSHOT</version>
     </parent>
-
     <groupId>it.geosolutions.mapstore</groupId>
     <artifactId>mapstore-product</artifactId>
     <packaging>war</packaging>
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <tomcat.version>8.5.69</tomcat.version>
+        <tomcat.port>8080</tomcat.port>
+        <datadir.location>${env.MAPSTORE_DATA_DIR}</datadir.location>
+        <security.integration>default</security.integration>
```

Comment:
shouldn't this be picked up from env aswell ?

Reply:
I'd like.
There is a limitation in assigning default values to properties when getting them from env variables that made it impossible. There is a solution, but uses profiles and it is a lot complicated. So I decided to keep it simple and allow it via `--` as the other properties. 

The datadir is more used and useful, so it could be useful to have also as an env variable.